contrib/99designs/gqlgen/tracer_test.go

```diff
@@ -140,6 +142,11 @@ func TestChildSpans(t *testing.T) {
 	for _, span := range allSpans {
 		if span.ParentID() == 0 {
 			root = span
+			assert.Equal("gqlgen", span.Tag("component"))
```

Comment:
It looks like you added Component and SpanKind to the ddtrace/ext/tags.go, but you don't use it everywhere. Please replace those string literals in libraries with a constant

Reply:
I disagree, I purposely left them out, because if someone were to change them in the future, this would force them to check tests to ensure that their change doesn't break anything. Otherwise people could change them without doing thorough checks of everything affected. This is only for tests that people often overlook

Reply:
If you check the tests though, they do use other constants

Reply:
Implemented